clicache/integration-test/DistOpsStepsN.cs

```diff
@@ -2328,7 +2317,7 @@ public virtual void IdictionaryRegionOperations(String RegionName)
       Util.Log("Idictionary_Item_Add_Get_Set_Remove_Step 4 complete.");
 
       // Chk with old api that element that is added by new api exists.
-      Assert.AreEqual(type.GetTypeItem<TValue>(ValueId, 0), region.Get(type.GetTypeItem<TKey>(KeyId, 0), null));
+      Assert.AreEqual(type.GetTypeItem<TValue>(ValueId, 0), region.Get(type.GetTypeItem<TKey>(KeyId, 0)));
```

Comment:
Do we still have tests that assert operations complete successfully when `null` is the callback value?

Reply:
yes